packages/dialtone-vue2/components/rich_text_editor/extensions/mentions/suggestion.js

```diff
@@ -64,6 +66,7 @@ export default {
       },
 
       onExit () {
+        console.log('EXIT');
```

Comment:
I assume you committed this by mistake 😅

Reply:
haha yep, thanks